packages/jest-changed-files/src/git.js

```diff
@@ -11,37 +11,19 @@ import type {Path} from 'types/Config';
 import type {Options, SCMAdapter} from 'types/ChangedFiles';
 
 import path from 'path';
-import childProcess from 'child_process';
+import execa from 'execa';
 
 const findChangedFilesUsingCommand = async (
   args: Array<string>,
   cwd: Path,
-): Promise<Array<Path>> =>
-  new Promise((resolve, reject) => {
-    const child = childProcess.spawn('git', args, {cwd});
-    let stdout = '';
-    let stderr = '';
-    child.stdout.on('data', data => (stdout += data));
-    child.stderr.on('data', data => (stderr += data));
-    child.on('error', e => reject(e));
-    child.on('close', code => {
-      if (code === 0) {
-        stdout = stdout.trim();
-        if (stdout === '') {
-          resolve([]);
-        } else {
-          resolve(
-            stdout
-              .split('\n')
-              .filter(s => s !== '')
-              .map(changedPath => path.resolve(cwd, changedPath)),
-          );
-        }
-      } else {
-        reject(code + ': ' + stderr);
-      }
-    });
-  });
+): Promise<Array<Path>> => {
+  const result = await execa('git', args, {cwd});
+
+  return result.stdout
```

Comment:
the rejection is different (but imo better), see https://github.com/sindresorhus/execa/blob/0e780f8ce9d4360188380f56b41dd39247ebc69a/readme.md#usage